src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.System/Microsoft.PowerToys.Run.Plugin.System.csproj

```diff
@@ -50,6 +50,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\Wox.Infrastructure\Wox.Infrastructure.csproj" />
     <ProjectReference Include="..\..\Wox.Plugin\Wox.Plugin.csproj" />
+    <ProjectReference Include="..\Microsoft.Plugin.Indexer\Microsoft.Plugin.Indexer.csproj" />
```

Comment:
Remove this reference. Plugins must not have cross-references.

Reply:
Not sure how that happened, but it has been removed.

Reply:
Figured out how that happened.  :)